docs/uperf.md

```diff
@@ -56,6 +57,8 @@ spec:
 
 `serviceip` will place the uperf server behind a K8s [Service](https://kubernetes.io/docs/concepts/services-networking/service/)
 
+`runtime_class` If this is set, the benchmark-operator will apply the runtime_class to the podSpec runtimeClassName.
+
```

Comment:
Do you want to add the same note: for Kata containers like in the fio docs above (line 165/166)?